src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -40,6 +48,21 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
     }
 
     this._config = config;
+    this._cardSize = config._card_size;
```

Comment:
Why would we store this as an instance variable? We can just read it from the config ?